accesspoc/dips/models.py

```diff
@@ -154,6 +154,7 @@ class DIP(AbstractEsModel):
         verbose_name=_('collection'),
     )
     dc = models.OneToOneField(DublinCore, null=True, on_delete=models.SET_NULL)
+    import_in_progress = models.BooleanField(default=False)
```

Comment:
Isn't a new dip always in progress? i.e. `defaut=True`?

Also, what would happen if the task never completes? Is there a way to report errors to the user? Do we need the result backend?

Reply:
I set default to true initially and realized that it would require a data migration to set the existing DIPs to false, which made me realize that setting it to true by default is a wrong assumption as the import may not always happen on creation (in the future). I may also remove this field if I find a better solution for this with the results backend.